### ISSUE TITLE
Adding gdrive index.tfl to tinfoil

### ISSUE THREAD

**Issue by yongbi85** [eXhumer/pyTinGen#13]
Sorry but can't seem to make it work...
I generated the index.tfl correctly and i tried adding the gdrive link directly to tinfoil with the output specified in the log
In tinfoil locations.conf I added gdrive:/xxxxXXXXXxxxxxx#index.tfl
(xxxxxxxx being the file id)
But it doesn't list anything

If I manually download the file and save it on the SD card and add the folder of the SD card where it is stored in the scan ==> Success i even get the message i've set up to be certain
If I add a gdrive folder in which i only put the index.tfl ==> fail, it doesn't list anything but if I click on it in file browser, i can see the content of the gdrive folder i.e index.tfl but still doesnt list...

So to sum it up
- gdrive file id directly added with #index.tfl to locations.conf ==> FAIL
-gdrive folder added to locations.conf ==> FAIL
-download index.tfl on sd card and add the sd folder to locations.conf ==> SUCCESS

Any help appreciated, maybe i do it wrong in the locations.conf...

{"url":"gdrive:/xxxxxxxxxxxxx#index.tfl","title":"gdrive perso","enabled"1}

Edit : before the question arised, yes i have access to the gdrive within tinfoil (copied credentials and token and using sxos)

**Comment by eXhumer** [comment]
I am aware of this. I will talk to blawar (tinfoil developer) to see if there is any direct way to do this properly.

**Comment by yongbi85** [comment]
Thanks for your work. I have done a docker image to make it run once a day on my synology

**Comment by kellylawrence** [comment]
@eXhumer Running into the same issue myself. I saw that you recently updated the `share_file` function to output an https based Google Drive URL to add to Tinfoil. Just wondering if that's still a work in progress? I updated my repo and ran the script again with:

```
--share-uploaded-index
--upload-to-my-drive
```

And while the file does get uploaded to my Google Drive, is set to appropriate sharing settings, and the new URL is added to Tinfoil's `locations.conf`, selecting that source under Tinfoil's "File Browser" tab result in no listings. I don't see New Games/DLC/Update tabs either, if that is set to the only index.

Adding that same file to the root of my SD card works fine though.

**Comment by eXhumer** [comment]
You must use tiny.cc and shorten the provided link apparently to use it with Tinfoil, as confirmed by Tinfoil developer.
![image](https://user-images.githubusercontent.com/62310242/94380831-015bb000-00ec-11eb-830a-d33ed95fd04a.png)

**Comment by eXhumer** [comment]
If you have a custom domain, you can set it to redirect to the `drive.google.com/uc?id=FILE_ID` and that will also work.

**Comment by yongbi85** [comment]
Yeah, thanks a lot for this tip, i did just that and so it is fully automatized now, refreshing the index every night and redirection in place so nothing to do on tinfoil side.
